README.md

```diff
@@ -52,22 +52,22 @@ Now, you can run the pipeline using:
 
 ```bash
 nextflow run nf-core/fastqrepair \
-   -profile <docker/singularity/.../institute> \
+   -profile <test/docker> \
    --input samplesheet.csv \
    --outdir <OUTDIR>
 ```
 
 optional parameters are:
 
-```txt
---chunk_size <int multiple of 4>
---qin <33/64>
---alphabet <ACGTN>
+```bash
+--chunk_size  <int multiple of 4>
+--qin         <33/64>
+--alphabet    <ACGTN>
 ```
```

Comment:
best to have parameter usage in the parameters docs, it's too easy to have things out of sync otherwise.

Reply:
They should be also there. Do you suggest to remove from README.md to avoid having to update both at any time?

Reply:
Exactly